content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -106,6 +106,29 @@ The UI to upload these certificates looks like:
 
 ![ptfe-ca-ui](./assets/ptfe-ca-bundle.png)
 
+
+## Alternative Terraform worker image
+
+TFE runs terraform plan and apply operations in a disposable docker container. There are cases where users may require additional tools that are not available in the default docker image. To work around this limitation users can build their own image and configure TFE to use that instead. In order for this to happen the name of the alternative docker image must be set in the config by using the relevant field as shown below:
+
+![ptfe-docker-image](./assets/ptfe-docker-image.png)
+
+During the first iteration of this feature it is important to impose certain limitations on what we can support. These limitations are:
+ - The base image must be ubuntu:xenial
+ - The user is responsible for making sure the image exists on the PTFE host by running `docker pull` from their own registry, or any other similar method.
+ - The user is responsible for making sure CA certificates are available when terraform runs.
```

Comment:
Let's say a bit more here - even more than the example below. I'd suggest that we explain which certs are likely to be needed and how we recommend injecting them.

Reply:
I added a small sentence explaining what we expect the file to contain. If we want to expand on what a CA cert is and which CAs we expect customers to include then this should be done on a different page, since this is also relevant to other configuration settings as well.